.github/workflows/cicd-2-publish.yaml

```diff
@@ -1,16 +1,17 @@
 name: "CI/CD publish"
 
 on:
-  pull_request:
-    types: [closed]
-    branches:
-      - main
+  push:
+    branches: [ "main" ]
+  # pull_request:
```

Comment:
where these are commented out could we just delete the lines - they will still exist in git history if we need to refer back to them